helm/kanister-operator/values.yaml

```
@@ -5,6 +5,11 @@ image:
   repository: ghcr.io/kanisterio/controller
   tag: 0.89.0
   pullPolicy: IfNotPresent
+repositoryServerControllerImage:
+  registry: ghcr.io/kanisterio
+  name: repo-server-controller
+  tag: 0.88.0
```

```suggestion
  tag: 0.89.0
```
Or whichever is latest at the time of merging.

---

will update this with latest when we will merge this PR